### PR TITLE
EN-2220 Renaming digital exchange items from plugins to components.

### PR DESCRIPTION
**Pull request by otarza**
https://jira.entando.org/browse/EN-2220
https://jira.entando.org/browse/EN-2253

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: test/test/mocks/digital-exchange/components.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// LIST_DE_PLUGINS_OK - List Digital Exchange plugins.
-export const LIST_DE_PLUGINS_OK = [
+// LIST_DE_COMPONENTS_OK - List Digital Exchange components.
```
Comment (nicpuddu) [inline test/test/mocks/digital-exchange/components.js]: let's avoid unnecessary comments. if a comment is necessary to explain the name of the variable it means that the name needs improving.

**Comment by entando-service** [comment]
All is well

File: src/api/digital-exchange/components.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { LIST_DE_COMPONENTS_OK } from 'test/mocks/digital-exchange/components';
+import { makeRequest, METHODS } from '@entando/apimanager';
+
+// eslint-disable-next-line
```
Comment (nicpuddu) [inline src/api/digital-exchange/components.js]: please do not use blanket disable statements: specify which rule you are disabling

File: test/test/mocks/digital-exchange/components.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
...ME1zHWhlVl5X0s0YMJUQ2vi9qASmRD8HLTAUrfUNt5wPtPCHJJPjlduWgJzCxERNPYrdTGzdyc5Y6sdL3rvFOi/IfPZS1Ql2ng7cYKclopYhvO7WLZZlgzFTgV5CZXw5OAMiUxLwzDw4eTLUkgSEDSXWbPlkqONsk21cHDLoKASOslhy4E8AURE3GSEgNWLt1DxUvZlG1PIvj34pCFMYAMj/XFACLsEJlF5dfPFCFUIJyG8ILy5G9hYKAxa3RNawnaTgz4HZguEYeFNlFGJL9FGRVHKqItWklIfLT4rIvs2vzzQI7qKA8PJSdAAytN9qxYiVjmKUr4I64qWwycKmQciVCi6uY8KC6TttPbilZoO4WzaefDV4XBwksLgJJ39sJAVJGW9ZJzehCwMVG+mEG5aeFi2aDSpN2OiCYms8IZITFsuzE87IinERjiUicyaGMUKgElwLBmILcYmihH+CcnFz2Wqci3QAhhMIeEqKEli7Nx2q5b4NopBIFG6utKGfWUDb9jK5Nr0EgBNlEIZEQYXJW5NESdEf2GSIbTGchpOpUCnOUMWZMDrTaA9pUHHIGnKs9ZodSNCAYAMmnJUBSFulPqEPkzSATokPgH60ockTLvbHJs/wAkwySt8Ns1f/rYFSVujpfWMNiOJbGZgeFujqUSrxLI5niuh80ctY+ifdNTevktdg1adoAMEyfmmibG4Stc4aTeaisq23FrUlMVCFwWR2sVnXMJbLFQd3wSapjUqhcso9WCeYK6ZCNt05iWOKAMADR2UDYmgkSkmNESt2WtsqxZndLixavb0J8EiVDkfSnlq8WKEmwYPcGlegRyLNuPWgME04oplli16atp+ZNxyli3SiWMAWBBYMFMLVI2OCdV5yVw9KAMAUQtdLtNb1stCZ+KFZbRBLlivhf4rDZNG6FpPT0WC40e1CIAGCOGHFNbwEO4AxDnrThJQGEdFOakYbpMzqW+b1ygEXuZbwZ4KgcQOSY2HWgQkSyNlh2lXu0MoIpOEm3FD+bgIDs8tbGyeYGCeCWCk3DCLSSI8iaoCGDBajSiCCmMrBK8xakslomghF/FEEgECwAY7AB0U3QBABiGCGkv+vVwm3XdXK2+EkDHSWO9KQblj3Ce8KTwtMIogojiC1NA2TzUgl73b9aJQBwBAXpKJUlgbxd5fdpwLB8NpkniiBgAIHmnNf/Z',
```
Comment (nicpuddu) [inline test/test/mocks/digital-exchange/components.js]: please don't use images for each record, we also need to test how it behaves if the value is either empty string or null.

**Comment by entando-service** [comment]
All is well
